app/src/functions/notify-endpoint-usage/app.ts

```diff
@@ -1,25 +1,28 @@
 import { influx, buildAppUsageQuery } from "../../lib/influx";
 import { getUTCTimestamp, getHoursFromNowUtcDate } from "../../lib/date-utils";
 import connect from "../../lib/db"
-import { ApplicationData, ExtendedApplicationData, ExtendedLoadBalancerData, ExtendedLoadBalancer, GetUsageDataQuery } from "../../models/types";
+import { ApplicationData, ExtendedApplicationData, ExtendedLoadBalancer, ExtendedLoadBalancerData, GetUsageDataQuery } from "../../models/types";
 import User from '../../models/User'
-import { getApplicationNetworkData, getAppsInNetwork } from "../../lib/pocket";
-import { QueryAppResponse, Application } from '@pokt-network/pocket-js';
+import { getAppsInNetwork } from "../../lib/pocket";
+import { Application } from '@pokt-network/pocket-js';
 import ApplicationModel, { IApplication } from "../../models/Application";
 import LoadBalancerModel, { ILoadBalancer } from "../../models/LoadBalancer";
+import Redis from 'ioredis'
 import { retryEvery } from "../../utils/retry";
 
 const redisHost = process.env.REDIS_HOST || "";
 const redisPort = process.env.REDIS_PORT || "";
 
 const maxRetries = process.env.MAX_RETRIES || 3;
 
+const redis = new Redis(parseInt(redisPort), redisHost)
+
 const calculateRelaysPercentage = (relays: number, maxRelays: number) => parseFloat(((relays / maxRelays) * 100).toFixed(2))
 
 export async function getUsageData(): Promise<GetUsageDataQuery[]> {
   const usage = (await influx.collectRows(
     buildAppUsageQuery({
-      start: getHoursFromNowUtcDate(1),
+      start: getHoursFromNowUtcDate(2),
```

Comment:
We should actually move this number into a constant above like `INFLUX_QUERY_START_TIME` and this should also be 0 I believe.